fem/bilinearform.cpp

```diff
@@ -1064,6 +1064,19 @@ void BilinearForm::Mult(const Vector &x, Vector &y) const
    }
 }
 
+void BilinearForm::MultTranspose(const Vector & x, Vector & y) const
+{
+   if (ext)
+   {
+      ext->MultTranspose(x, y);
+   }
+   else
+   {
+      y = 0.0;
```

Comment:
Shouldn't this initialization be shared for both if-else-statements?
I see also that all `PAConvectionApplyT` but the `Smem` `3D` use `+=`: is it intended?

Reply:
Good catch! This indeed was a bug, it also was in `SmemPAConvectionApply3D`.

Though, `BilinearFormExtension` only has `Mult` and `MultTranspose` (not `AddMult` and `AddMultTranspose`), so I think this is correct.